presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -1223,6 +1223,16 @@ public void testLambda()
         assertQuery("select transform(x, i->transform(i, j->j*y)) from (select x, y*y as y from (values row(array[array[1]], 2)) t(x, y))");
     }
 
+    @Test
+    public void testCDF()
+    {
+        assertQuery("SELECT binomial_cdf(CAST (nationkey AS INTEGER), 0.1, 0 ) FROM nation WHERE nationkey > 0");
```

Comment:
Please can you check if these CASTs are necessary on the Presto-java side also ( i.e. Does the query without the casting work ?)

Reply:
@aditi-pandit CASTs are necessary, checked on the Java side.